backend/api/v1/messages.file.service.js

```diff
@@ -2,24 +2,24 @@
 const fs = require('fs');
 const status = require('http-status');
 
-const writeMessagesToJsonFile = (res, messages, path) => {
+const writeMessagesToJsonFile = (res, messages, path, statusCode) => {
```

Comment:
why do you wanna the right status code in the file?

Reply:
I've noticed that if POST message is preformed usually status.created = 201 is sent, and put/delete return status.ok = 200, that's why i've decided to change it and send this statusCode as an argument, I'm i wrong?